src/platforms/weex/runtime/modules/class.js

```diff
@@ -44,6 +44,8 @@ function makeClassList (data: VNodeData): Array<string> {
     classList.push.apply(classList, dataClass)
   } else if (isObject(dataClass)) {
     classList.push.apply(classList, Object.keys(dataClass).filter(className => dataClass[className]))
+  } else if (typeof dataClass === 'string') {
+    classList.push.apply(classList, [dataClass])
```

Comment:
What if the `dataClass` is a class list? such as `"foo bar"`. It should be splited by white spaces.